lib/matplotlib/cbook/__init__.py

```diff
@@ -1649,7 +1650,7 @@ def index_of(y):
        The x and y values to plot.
     """
     try:
-        return y.index.values, y.values
```

Comment:
Do we need both this change and the additional exception handling above?

Reply:
I don't know, but I think we should be in the habit of using `to_numpy()` when possible?

Reply:
`to_numpy` came in via pandan 0.24 on Jan 25, 2019 so we can rely on it being there and https://stackoverflow.com/a/54508052/380231 makes an arguement in favor of `no_numpy`.

I think this change may be un-related (or fix this by chance) but I do not think it will avoid needing the other one and does no harm.